app/views/supervisor/dashboard/index.html.slim

```diff
@@ -2,29 +2,44 @@ h2 Supervisor Dashboard
 h5 Alpha Version
 
 div.row
-  div.dashboard-box.col-sm-12
+  div.dashboard-box.col-xs-12
     div.background-color
       section.team-title
         h3 Overview
         h5 Under Construction
       section.dashboard
-        p = link_to "Supervisors on GitHub", 'https://github.com/rails-girls-summer-of-code/supervisors'
-        p = link_to "Guide", 'https://github.com/rails-girls-summer-of-code/supervisors/blob/master/guide.md'
-        /this is where all the aggregated messages and info should go
-        /= flash[:notice]
+        div.row
+          div.links.col-sm-6 class=("col-lg-8" if supervised_teams.count > 2)
+            p = link_to "Guide", 'https://github.com/rails-girls-summer-of-code/supervisors/blob/master/guide.md'
+            p = link_to "Supervisors on GitHub", 'https://github.com/rails-girls-summer-of-code/supervisors'
+            table.legend
+              tr
+                td
+                  .performance-icon = render 'green'
+                td
+                  '= Alive and kicking
+              tr
+                td
+                  .performance-icon = render 'orange'
+                td
+                  '= Needs TLC
+              tr
+                td
+                  .performance-icon = render 'red'
+                td
+                  '= Red alert
```

Comment:
We are Cheesy and we wanted to show that :-) We did a hallway usability test, with the icons plus this cryptic explanation, and that worked. Let's ask our Usability Expert for a verdict.